### ISSUE TITLE
Upgrade clang-format version from 6.0.0 to 7.0.0

### ISSUE THREAD

**Issue by soleo** [git-up/GitUp#424]
The current version of clang-format from homebrew is `7.0.0` now. 

```
$ clang-format -version
clang-format version 7.0.0 (tags/google/stable/2018-01-11)
```

**Comment by swisspol** [comment]
Fixed by f5de52030a174e7bbeb35adefcb10ae5d08b54f7

**Comment by soleo** [comment]
Thanks for the quick fix.
